apps/gateway/src/routes/v2/documents/logs/create.handler.ts

```diff
@@ -0,0 +1,40 @@
+import { LogSources } from '@latitude-data/core/browser'
+import { generateUUIDIdentifier } from '@latitude-data/core/lib/generateUUID'
+import { createDocumentLog } from '@latitude-data/core/services/documentLogs/create'
+import { AppRouteHandler } from '$/openApi/types'
+import { CreateLogRoute } from '$/routes/v2/documents/logs/create.route'
+import { getData } from '$/common/documents/getData'
+
+// @ts-expect-error: broken types
```

Comment:
i mean could we fix types 😅 it looks like this PR is breaking all types

Reply:
Before we were not using openapi so no old types broken. Just this
```
AppRouteHandler<CreateLogRoute>
```
And is because it does a complex generic black magic. But types inside the handlers all work nicely